src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -407,6 +490,7 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
     restoreSparkConf()
     executeModules()
     buildFacts()
+    verboseAudit()
```

Comment:
This should not be here -- when you follow the pattern this logic will become part of refreshViews. 

Just add case to the very end and make a complex case statement
case audit + notebooks + clusterEvents --> publish this view

...
case audit & notebooks & clusterEvents => {GoldTargets.viewTarget.publish(...)}

The above is obviously pseudocode -- you may have to do a logical operator func and use if for this case